tests/sql/moz-fx-data-shared-prod/org_mozilla_fenix_nightly_derived/geckoview_version_v1/metrics.schema.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "type": "TIMESTAMP",
+    "name": "submission_timetamp",
+    "mode": "REQUIRED"
+  },
+  {
+    "type": "RECORD",
+    "name": "client_info",
+    "mode": "NULLABLE",
+    "fields": [
+      {
+        "type:": "STRING",
+        "name": "app_build",
+        "mode": "NULLABLE"
+      }
+    ]
+  },
+  {
+    "type": "RECORD",
+    "name": "metrics",
+    "mode": "NULLABLE",
+    "fields": [
+      {
+        "type": "RECORD",
+        "name": "string",
+        "mode": "NULLABLE",
+        "fields": [
+          {
+            "type:": "STRING",
+            "name": "geckoview_version",
+            "mode": "NULLABLE"
+          }
+        ]
+      }
+    ]
+  }
+]
+  
```

Comment:
I've been trying to set up tests with the schema to cast the relevant fields to string, but I get the following errors:

```python
>>> from google.cloud import bigquery

# works fine
>>> bq.schema_from_json("tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_aggregates_v1/test_aggregation/mobile_search_clients_daily_v1.schema.json")

>>> bq.schema_from_json("tests/sql/moz-fx-data-shared-prod/org_mozilla_fenix_nightly_derived/geckoview_version_v1/metrics.schema.json")

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/client.py", line 3166, in schema_from_json
    return self._schema_from_json_file_object(file_obj)
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/client.py", line 3147, in _schema_from_json_file_object
    return [SchemaField.from_api_repr(field) for field in json_data]
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/client.py", line 3147, in <listcomp>
    return [SchemaField.from_api_repr(field) for field in json_data]
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/schema.py", line 105, in from_api_repr
    fields=[cls.from_api_repr(f) for f in fields],
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/schema.py", line 105, in <listcomp>
    fields=[cls.from_api_repr(f) for f in fields],
  File "/Users/amiyaguchi/Work/bigquery-etl/venv/lib/python3.8/site-packages/google/cloud/bigquery/schema.py", line 104, in from_api_repr
    field_type=api_repr["type"].upper(),
KeyError: 'type'
```

@relud would you happen to know where I've gone wrong with this schema? I tried this out in the BigQuery console, and it seems to create a new table fine.